Examples/Liquidate/run.sh

```diff
@@ -0,0 +1,7 @@
+certoraRun BorrowSystem.sol:SimpleBorrowSystem DummyERC20A.sol DummyERC20B.sol \
+	--link SimpleBorrowSystem:collateralToken=DummyERC20A SimpleBorrowSystem:borrowToken=DummyERC20B \
+	--verify SimpleBorrowSystem:BorrowSystem.spec \
+	--solc solc8.4 \
+	--settings -postProcessCounterExamples=true,-assumeUnwindCond \
```

Comment:
We should use `--optimistic_loops` instead of `--settings -assumeUnwindCondition`

Reply:
fixed